bot/exts/moderation/incidents.py

```diff
@@ -113,9 +121,9 @@ def is_incident(message: discord.Message) -> bool:
     """True if `message` qualifies as an incident, False otherwise."""
     conditions = (
         message.channel.id == Channels.incidents,  # Message sent in #incidents
-        not message.author.bot,                    # Not by a bot
-        not message.content.startswith("#"),       # Doesn't start with a hash
-        not message.pinned,                        # And isn't header
+        not message.author.bot,  # Not by a bot
+        not message.content.startswith("#"),  # Doesn't start with a hash
+        not message.pinned,  # And isn't header
```

Comment:
I'm not sure why you removed the spaces here, they were added for readability

Reply:
You marked the conversation as resolved, have you reverted this?

Reply:
> You marked the conversation as resolved, have you reverted this?

It was resolved and the change had been done, but sometimes PyCharm pops up a message of `Reformat code`, and on clicking that, this change happens.

I must have missed this piece of code while going through the code before committing.

Reply:
Resolved in commit 93f9243.

Reply:
This should still be reverted